sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -3062,6 +3080,9 @@ object Sequence {
         "org.apache.spark.sql.catalyst.util.DateTimeUtils.timestampNTZAddInterval"
     }
 
+    private val daysToMicrosCode = "org.apache.spark.sql.catalyst.util.DateTimeUtils.daysToMicros"
+    private val microsToDaysCode = "org.apache.spark.sql.catalyst.util.DateTimeUtils.microsToDays"
+
```

Comment:
Could you introduce a val for DateTimeUtils:
```scala
    private val dtu = DateTimeUtils.getClass.getName.stripSuffix("$")
```
and re-use it here (and above):
```scala
    private val addIntervalCode = outerDataType match {
      case TimestampType | DateType => s"$dtu.timestampAddInterval"
      case TimestampNTZType => s"$dtu.timestampNTZAddInterval"
    }

    private val daysToMicrosCode = s"$dtu.daysToMicros"
    private val microsToDaysCode = s"$dtu.microsToDays"
```
Look at datetimeExpressions.scala for examples.